packages/yarnpkg-core/sources/StreamReport.ts

```diff
@@ -438,12 +406,20 @@ export class StreamReport extends Report {
     const prefix = formattedName ? `${formattedName}: ` : ``;
 
     if (!this.json) {
-      this.writeLineWithForgettableReset(`${this.formatPrefix(prefix, `redBright`)}${text}`, {truncate: false});
+      this.writeLine(`${this.formatPrefix(prefix, `redBright`)}${text}`, {truncate: false});
     } else {
       this.reportJson({type: `error`, name, displayName: this.formatName(name), indent: this.formatIndent(), data: text});
     }
   }
 
+  reportFold(title: string, text: string) {
```

Comment:
I considered it (it was originally like that), but it didn't seem much cleaner: a question you quickly reach is whether the text getter should be sync or async. Using sync prevents things like `readFilePromise`, whereas async feels awkward in an API otherwise exclusively sync.

I think the check is fine for now.

> (Let's also make sure that none of that is executed during the benchmarks which are run on GitHub Actions)

They only execute when an error happens, which shouldn't happen during benchmarks.

Reply:
> They only execute when an error happens, which shouldn't happen during benchmarks.

True, I was referring to it in general, since for example we also report groups for timers on CI that should technically be reported during the benchmarks too :thinking: